Add new repo: ontove ontology

AlessandroBertozzi opened the pull request.

TallTed commented on ontove/.htaccess:
```diff
@@ -0,0 +1,58 @@
+# di Spagna in Bologna: with this digital catalogue, built under the guidance of Professors Domenico 
+# Maffei and Andrea Padovani, not only can the fragile codices be readily accessed and studied, but their 
+# preservation for future use is guaranteed, too. The project - named for the primus illuminator of 
+# legal science studies in Bologna, the Italian jurist Irnerius - have been possible by the Collegio di Spagna that 
```
```suggestion
# legal science studies in Bologna, the Italian jurist Irnerius - have been made possible by the Collegio di Spagna that 
```

TallTed commented on ontove/README.md:
```diff
@@ -0,0 +1,6 @@
+The OntoVE project, Ontologies for the valorization of Germanic Cultural Heritage in Veneto within the perspective of the European Open Science Cloud (EOSC), has as its goal the development of  ontological modules for the description of Germanic Cultural Heritage (CH) items related to the Veneto region and provide different visualization outputs designed for both an academic and a non-academic au ...
```
```suggestion
The OntoVE project, Ontologies for the valorization of Germanic Cultural Heritage in Veneto within the perspective of the European Open Science Cloud (EOSC), has as its goal the development of ontological modules for the description of Germanic Cultural Heritage (CH) items related to the Veneto region and provision of different visualization outputs designed for both academic and a non-academic audiences.
```

TallTed commented on ontove/README.md:
```diff
@@ -0,0 +1,6 @@
+The OntoVE project, Ontologies for the valorization of Germanic Cultural Heritage in Veneto within the perspective of the European Open Science Cloud (EOSC), has as its goal the development of  ontological modules for the description of Germanic Cultural Heritage (CH) items related to the Veneto region and provide different visualization outputs designed for both an academic and a non-academic au ...
+The ultimate aim of the project is to connect heterogeneous data, such as archaeological findings or written documents, and provide uniform descriptions of them. The dataset is constituted by CH items of Germanic origin linked to Venice and its neighbouring territories. 
+In order to capture the defining features of these items, established ontologies are reused. The module devised for the description of CH items is called OntoVE_Archeo and reuses the top-level ontology CIDOC-CRM, v. 7.1.2 (Bekiari et al. 2022) and the finer-grained ontology ArCo-Architettura della Conoscenza, developed by Carriero et al. (2019) in collaboration with the ICCD (http://www.iccd.beni ...
+The OntoVE_WrittenRecords module has as its centre the semantic description of bibliographic items; it reuses LRMoo v 0.9.6 (Bekiari et al. 2023), Bibframe (https://www.loc.gov/bibframe/implementation/) and the Bibo (D’Arcus/Giasson 2016, https://www.dublincore.org/specifications/bibo/) ontology. The modules where developed with Protégé Desktop, version 5.5.0 (http://protege.stanford.edu, Musen 2 ...
```
```suggestion
The OntoVE_WrittenRecords module has as its centre the semantic description of bibliographic items; it reuses LRMoo v 0.9.6 (Bekiari et al. 2023), Bibframe (https://www.loc.gov/bibframe/implementation/), and the Bibo (D’Arcus/Giasson 2016, https://www.dublincore.org/specifications/bibo/) ontology. The modules where developed with Protégé Desktop, version 5.5.0 (http://protege.stanford.edu, Musen 2015). 
```

TallTed commented on ontove/README.md:
```diff
@@ -0,0 +1,6 @@
+The ultimate aim of the project is to connect heterogeneous data, such as archaeological findings or written documents, and provide uniform descriptions of them. The dataset is constituted by CH items of Germanic origin linked to Venice and its neighbouring territories. 
+In order to capture the defining features of these items, established ontologies are reused. The module devised for the description of CH items is called OntoVE_Archeo and reuses the top-level ontology CIDOC-CRM, v. 7.1.2 (Bekiari et al. 2022) and the finer-grained ontology ArCo-Architettura della Conoscenza, developed by Carriero et al. (2019) in collaboration with the ICCD (http://www.iccd.beni ...
+The OntoVE_WrittenRecords module has as its centre the semantic description of bibliographic items; it reuses LRMoo v 0.9.6 (Bekiari et al. 2023), Bibframe (https://www.loc.gov/bibframe/implementation/) and the Bibo (D’Arcus/Giasson 2016, https://www.dublincore.org/specifications/bibo/) ontology. The modules where developed with Protégé Desktop, version 5.5.0 (http://protege.stanford.edu, Musen 2 ...
+Both modules align semantically equivalent classes and assign classes from finer-grained ontologies as subclasses of the more general and abstract classes provided by CIDOC-CRM and its interoperable bibliographic module LRMoo. By semantic alignment operations and the reuse of established ontologies the project adheres to the FAIR principles (Kush et al. 2020).
```
```suggestion
Both modules align semantically equivalent classes and assign classes from finer-grained ontologies as subclasses of the more general and abstract classes provided by CIDOC-CRM and its interoperable bibliographic module LRMoo. By semantic alignment operations and the reuse of established ontologies, the project adheres to the FAIR principles (Kush et al. 2020).
```

TallTed reviewed (changes requested):
Small grammar and punctuation fixes

dgarijo commented:
@AlessandroBertozzi please review suggestions and please clarify who the maintainer for this project will be. As stated in the htaccess it is:
```
# Contacts:
# - Monica Palmirani <monica.palmirani@unibo.it>
# - Silvio Peroni <silvio.peroni@unibo.it>
```
Is this correct? Please add yourself as a maintainer in the readme too

AlessandroBertozzi commented:
I added the contacts and made the suggested changes to the texts. Sorry about the little mishap on the .htaccess. It was a text left over from an old project. I corrected and aligned it
